scipy/ndimage/tests/test_fourier.py

```diff
@@ -129,3 +130,16 @@ def test_fourier_ellipsoid_1d_complex(self):
                 a = ndimage.fourier_ellipsoid(x, 5, -1, 0)
                 b = ndimage.fourier_uniform(x, 5, -1, 0)
                 assert_array_almost_equal(a, b, decimal=dec)
+
+    @pytest.mark.parametrize('shape', [(0, ), (0, 10), (10, 0)])
+    @pytest.mark.parametrize('dtype, dec',
+                             [(numpy.float32, 5), (numpy.float64, 14),
+                              (numpy.complex64, 5), (numpy.complex128, 14)])
+    @pytest.mark.parametrize('test_func',
+                             [ndimage.fourier_ellipsoid,
+                              ndimage.fourier_gaussian,
+                              ndimage.fourier_uniform])
+    def test_fourier_zero_length_dims(self, shape, dtype, dec, test_func):
```

Comment:
Nit: The `dec` argument isn't being used.
```suggestion
    @pytest.mark.parametrize('dtype',
                             [numpy.float32, numpy.float64,
                              numpy.complex64, numpy.complex128])
    @pytest.mark.parametrize('test_func',
                             [ndimage.fourier_ellipsoid,
                              ndimage.fourier_gaussian,
                              ndimage.fourier_uniform])
    def test_fourier_zero_length_dims(self, shape, dtype, test_func):
```

Reply:
@peterbell10 `dec` argument removed, thanks for spotting.